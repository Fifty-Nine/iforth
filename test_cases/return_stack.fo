1 >r .d rclear
2 >r r> .d clear
3 >r r@ .d clear rclear
4 >r rdrop .d
