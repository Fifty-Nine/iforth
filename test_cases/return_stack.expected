========= machine state =========
token stream:
0:[1] 1:[>r] 2:[.d] 3:[rclear] 4:[2] 5:[>r] 6:[r>] 7:[.d] 8:[clear] 9:[3] 10:[>r] 11:[r@] 12:[.d] 13:[clear] 14:[rclear] 15:[4] 16:[>r] 17:[rdrop] 18:[.d] 

data stack:
[]

return stack:
[0:1]

ip: 2 (.d)
=================================
========= machine state =========
token stream:
0:[1] 1:[>r] 2:[.d] 3:[rclear] 4:[2] 5:[>r] 6:[r>] 7:[.d] 8:[clear] 9:[3] 10:[>r] 11:[r@] 12:[.d] 13:[clear] 14:[rclear] 15:[4] 16:[>r] 17:[rdrop] 18:[.d] 

data stack:
[0:2]

return stack:
[]

ip: 7 (.d)
=================================
========= machine state =========
token stream:
0:[1] 1:[>r] 2:[.d] 3:[rclear] 4:[2] 5:[>r] 6:[r>] 7:[.d] 8:[clear] 9:[3] 10:[>r] 11:[r@] 12:[.d] 13:[clear] 14:[rclear] 15:[4] 16:[>r] 17:[rdrop] 18:[.d] 

data stack:
[0:3]

return stack:
[0:3]

ip: 12 (.d)
=================================
========= machine state =========
token stream:
0:[1] 1:[>r] 2:[.d] 3:[rclear] 4:[2] 5:[>r] 6:[r>] 7:[.d] 8:[clear] 9:[3] 10:[>r] 11:[r@] 12:[.d] 13:[clear] 14:[rclear] 15:[4] 16:[>r] 17:[rdrop] 18:[.d] 

data stack:
[]

return stack:
[]

ip: 18 (.d)
=================================
